sql-cli/sql_cli/macros/load_file.py.jinja2

```diff
@@ -0,0 +1,64 @@
+{% macro load_file(yaml_file, dag) -%}
+{% set yaml_content = yaml_file.get_yaml_content() -%}
+{{ yaml_file.get_variable_name() }} = aql.load_file(
+    {#- Note: Unfortunately, we cannot simply use **kwargs as some of the deserialized fields
```

Comment:
Create a ticket to evaluate if we can support **kwargs with a combination of python helper methods to tranform the yaml content as is needed by Jinja templates.

Reply:
Created ticket: #1284